library/x509_crt.c

```diff
@@ -1126,6 +1139,7 @@ int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path )
     char filename[MAX_PATH];
     char *p;
     size_t len = strlen( path );
+    int lengthAsInt = 0;
```

Comment:
I agree with Darryl's comment - this name doesn't follow the Mbed TLS conding standard.

Reply:
Fixed.